nums/numpy/api.py

```diff
@@ -93,7 +93,7 @@ def wrapped(*args, **kwargs):
 float16 = np.float16
 float32 = np.float32
 float64 = np.float64
-float128 = np.float128
+#float128 = np.float128
```

Comment:
Why is this commented out?

Reply:
Actually, this was causing an error on my system. I had posted about this on the Slack and Mohamed said that just comment it out for now as he had added that. I will add it back.